src/System.Reflection.Context/tests/CustomReflectionContextTests.cs

```diff
@@ -9,6 +9,7 @@ namespace System.Reflection.Context
     public class CustomReflectionContextTests
     {
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
Yes, and that is why we are throwing Platform not supported exception. @AtsushiKan can confirm too.